### ISSUE TITLE
Suggestion: Add option to display first note of Melody Playback

### ISSUE THREAD

**Issue by decaffjoe** [leafo/sightreading.training#115]
# Rationale

TLDR; give users the option to focus on relative pitch training vs perfect pitch training in Melody Playback

Right now I find myself ignoring the melody until I can figure out what the first note is. After I've figured out it's D5 for instance, then I can focus on the intervals. This is not ideal since I want to be spending all of my time focusing on the intervals instead of first having to train my perfect pitch.

An option to give away the first note would allow users to train either way.

# Implementation

See image below for rough idea of what this might look like.

I would love to implement this feature myself as a way to get familiar with the code base if we agree this feature is worth adding. The app is really awesome, thanks a lot for making it!

![image](https://user-images.githubusercontent.com/113792540/191823712-889d1cdb-b15f-40b3-bde2-14e248d4e292.png)

**Comment by leafo** [comment]
Good suggestion, added a quick toggle for it
